### ISSUE TITLE
has anyone found a way to make it so you can log in to your Google Play services on windows

### ISSUE THREAD

**Issue by 2chiefk** [microg/GmsCore#2721]
OK I've tried several different ways it's interesting I have no problems when I use some of the browser but every time I try like a Google Play services AG services like you got here it automatically errors out I have no idea has anybody got any clue what's going on here I tried the X64 versions I've tried the I've tried a lot of different ways anybody know what's going on I'd be loved to see some but he answered explained 'cause I can log into like YouTube if I use the YouTube tool APK does anybody remember that yeah that one still works One that works sometimes the Youtube vanced APK

**Comment by 2chiefk** [comment]
- And obviously you guys can email me back at you know Gmail.com yeah I can see the name it's the same exact 2chefk

**Comment by ale5000-git** [comment]
So, what is this?
Spam from an AI bot?

**Comment by dimaryaz** [comment]
Yeah, that user's activity is all fake. I just reported it to GitHub.

**Comment by 2chiefk** [comment]
> So, what is this?
> Spam from an AI bot?


What your a bot

**Comment by ale5000-git** [comment]
The message is completely incomprehensible.
Completely without punctuation.

No offense but it must be spam because even a 10 year old could write better.

**Comment by 2chiefk** [comment]
> So, what is this? Spam from an AI bot?

Actually by the way there is a way kind of got it resolved. You download games services and that gives you play services. I don't know why you all couldn't just have told me that. Yeah, I think I'm a freaking spam or a bar. What the f*** is wrong with you all. I designed this s*** as a kid. I'm not 50 okay

**Comment by 2chiefk** [comment]
Yeah there's something seriously wrong with you when a 50-year-old guy asks you guys something so simple that any of you could have told me and not think I'm a f****** bot or not. Think I'm a f****** goddamn m************ spam. All you had to say was you need to download the Google games and I would have figured it out what the f***

**Comment by 2chiefk** [comment]
And obviously I've bought and spare maker be f****** swearing at you okay f****** f***** remember that next time as long as you're talking to one of the guys that originally invented this s***

**Comment by 2chiefk** [comment]
One last thing. Dumbasses I'm not on here that often so don't expect me to reply to you really fast what the f***
